src/Label.js

```diff
@@ -43,6 +44,7 @@ const Label = (props) => {
     tag: Tag,
     check,
     inline,
+    disabled,
```

Comment:
should `disabled` be passed into the rendered component? I don't think so in this context, but it's usually something that would get passed through in something like an input.